.gitmodules

```diff
@@ -130,3 +130,7 @@
     ignore = dirty
     path = third_party/tensorpipe
     url = https://github.com/pytorch/tensorpipe.git
+[submodule "third_party/valgrind"]
+    ignore = dirty
+	path = third_party/valgrind
+	url = https://sourceware.org/git/valgrind.git
```

Comment:
lol the whitespace here

Reply:
VSCode inserts the last blank line whether I like it or not.